src/epics/sync.js

```diff
@@ -1,33 +1,27 @@
 import { Observable } from 'rxjs';
 import { combineEpics } from 'redux-observable';
-import { LOCATION_CHANGE } from 'react-router-redux';
+import { LOCATION_CHANGED } from 'redux-little-router';
 import {
 	apiRequest,
 	apiSuccess,
 	apiError,
 	apiComplete,
 	locationSync,
 } from '../actions/syncActionCreators';
-import { activeRouteQueries$ } from '../util/routeUtils';
+import { activeRouteQueries } from '../util/routeUtils';
 import { fetchQueries } from '../util/fetchUtils';
 
 /**
  * Navigation actions will provide the `location` as the payload, which this
- * epic will use to collect the current Reactive Queries associated with the
- * active routes.
+ * epic will use to collect the current Reactive Queries  *
  *
  * These queries will then be dispatched in the payload of `apiRequest`
- * @param {Object} routes The application's React Router routes
- * @returns {Function} an Epic function that emits an API_REQUEST action
  */
-export const getNavEpic = routes => {
-	const activeQueries$ = activeRouteQueries$(routes);
-	return (action$, store) =>
-		action$.ofType(LOCATION_CHANGE, '@@server/RENDER', 'LOCATION_SYNC')
-			.map(({ payload }) => payload)  // extract the `location` from the action payload
-			.flatMap(activeQueries$)        // find the queries for the location
-			.map(apiRequest);               // dispatch apiRequest with all queries
-};
+export const navEpic = (action$, store) =>
+	action$.ofType(LOCATION_CHANGED, 'LOCATION_SYNC')
```

Comment:
no - redux-little-router provides a way of manually triggering its `LOCATION_CHANGED` event with the initial location on the server, which is effectively what `@server/RENDER` was being used to signal.